platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -578,6 +582,18 @@ class EditorEditText : AppCompatEditText {
         )
     }
 
+    override fun addTextChangedListener(watcher: TextWatcher) {
+        if (!this::textWatcher.isInitialized) {
```

Comment:
Thanks @ganfra for this, I didn't really remember how to handle code that was called when a view is inflated and was stuck with a null pointer exception.